dm/proto/dmworker.proto

```diff
@@ -144,6 +146,7 @@ message ShardingGroup {
 
 // SyncStatus represents status for sync unit
 message SyncStatus {
+    // totalEvents/totalTps/recentTps has been deprecated now
     int64 totalEvents = 1;
     int64 totalTps = 2;
     int64 recentTps = 3;
```

Comment:
`Txx` and `Rxx` is calculated the same way, seems no need to rename them.

Reply:
It is a naming error

Reply:
better add those `Rxx` in another pr, those fields are exposed to user.

Reply:
yes, it will expose. So I also update the [document](https://github.com/pingcap/docs-cn/pull/11755)